monai/transforms/transform.py

```diff
@@ -213,6 +212,11 @@ class Transform(ABC):
         :py:class:`monai.transforms.Compose`
     """
 
+    backend: List[str] = []
+    """Transforms should add data types to this list if they are capable of performing a transform without
```

Comment:
the attribute docstring should be immediately after a simple assignment at the top level of a module. As I update this module I noticed that `Fourier` shouldn't be part of this base API file, so I moved it to transform utils.